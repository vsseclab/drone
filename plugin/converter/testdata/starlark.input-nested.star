def main(ctx):
  return {
    "kind": "pipeline",
    "name": "default",
    "steps": [
      ctx.input.builds
    ]
  }