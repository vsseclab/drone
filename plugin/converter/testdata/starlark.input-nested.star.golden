{"kind": "pipeline", "name": "default", "steps": [{"name": "build", "image": "mcr.microsoft.com/dotnet/sdk:5.0", "commands": ["dotnet build"]}]}